### PR TITLE
Rounding fix in cutout_tools.make_cutouts

### PR DESCRIPTION
**Pull request by robelgeda**
astroimtools.cutout_tools.make_cutouts uses reproject_interp and it keeps failing because of “shape_out=(y_pix, x_pix)” parameter. The y_pix and x_pix need to be an int not a float.

### CODE REVIEW

File: astroimtools/cutout_tools.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def make_cutouts(catalogname, imagename, image_label, apply_rotation=False,
 
             try:
                 cutout_arr = reproject_interp(
-                    (data, wcs), cutout_hdr, shape_out=(y_pix, x_pix), order=2)
+                    (data, wcs), cutout_hdr, shape_out=(int(np.ceil(y_pix)), int(np.ceil(x_pix))), order=2)
```
Comment (pllim) [inline astroimtools/cutout_tools.py]: Thanks. Is the `np.ceil` part really necessary?

Comment (philhodge) [inline astroimtools/cutout_tools.py]: If you want a `round` function that works the same in Python 2 and 3, you can use:

math.floor(x + math.copysign(0.5, x))

Comment (robelgeda) [inline astroimtools/cutout_tools.py]: I have made the correction.

**Comment by robelgeda** [comment]
Ha! @pllim , this is before we met in person 😄. I am not sure why travis is failing but we can merge it if it looks good to you. We will probably need it during next weeks sprint.

**Comment by pllim** [comment]
Did you rebase recently? If not, try that.

**Comment by robelgeda** [comment]
Finally giving it a go

**Comment by pllim** [comment]
Please fix flake8 failure. Thanks!

**Comment by pllim** [comment]
I wouldn't worry about the coverage report. Merging. Thanks!

**Pull request merged**